CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing to flv-kf-protocol
+
+Thank you for contributing. No matter how large or small, contributions are always welcome. Before contributing, please read [Code of Conduct](CODE-OF-CONDUCT.md)
+
+#### Table Of Contents
+
+[Assumptions](#assumptions)
+
+[Ask a Question](#ask-a-question)
+
+[Getting Started](#getting-started)
+
+[Contributing](#contributing)
+
+## Assumptions
+Familiarity with
+- [Rust](https://www.rust-lang.org)
+- [Kafka](https://kafka.apache.org/)
+- [Fluvio](https://www.fluvio.io/docs/)
```

Comment:
Can remove section about fluvio.  This project doesn't depend on fluvio, it's vice versa